databricks/koalas/frame.py

```diff
@@ -3333,7 +3334,7 @@ def to_table(self, name: str, format: Optional[str] = None, mode: str = 'error',
         self.to_spark().write.saveAsTable(name=name, format=format, mode=mode,
                                           partitionBy=partition_cols, **options)
```

Comment:
oh,, should this lines be `self.to_spark().write.mode(mode).saveAsTable(...)`?

Reply:
And ditto for other `to_xxx` related to Spark IO?

Reply:
Eh, do you mean for consistency? `saveAsTable` in Spark seems having `mode`.

Reply:
I just made the changes back.

Reply:
nvm, I thought we should explicitly specify `mode` with `write.mode(mode)`, but it's not needed.